### PR TITLE
test checkErrorMessage created

### PR DESCRIPTION
**Pull request by AnaLama**

### CODE REVIEW

File: src/test/java/GroupBughuntersTest.java [hunk 1]
```diff
@@ -252,5 +252,20 @@ public void testInsuranceCompanyQuote() throws InterruptedException {
         String registrationValue = getDriver().findElement(By.xpath("/html/body/table/tbody/tr[6]/td[2]")).getText();
         Assert.assertEquals(registrationValue,registration);
     }
+    @Test
```
Comment (eiliasov) [inline src/test/java/GroupBughuntersTest.java:255]: please add "test" in the beginning of method's name

**Review by eiliasov — COMMENTED** [review]
please add "test" in the name of the method

File: src/test/java/GroupBughuntersTest.java (hunk 1 above)
Comment (AnaLama) [reply src/test/java/GroupBughuntersTest.java:255]: My test is named testCheckErrorMessage. Where else to add "test"?

Comment (AnaLama) [reply src/test/java/GroupBughuntersTest.java:255]: oh, I see now. How come this word does not exist here? I added it before commiting and can see it in IDE. Does anybody know how to add it now?

**Review by eiliasov — APPROVED** [review]

**Review by bahasadyr — APPROVED** [review]

**Pull request merged**